pkg/skaffold/schema/latest/config.go

```diff
@@ -750,6 +750,10 @@ type BuildpackArtifact struct {
 	// Order matters.
 	Buildpacks []string `yaml:"buildpacks,omitempty"`
 
+	// ProjectDescriptor is the path to the toml project descriptor.
+	// Defaults to `project.toml`.
```

Comment:
I don't think the toml is necessary since we have it below.  And it is a file.  (This is similar what `pack` shows.)
```suggestion
	// ProjectDescriptor is the path to the project descriptor file.
	// Defaults to `project.toml` if it exists.
```